src/FirstSteps/RPG/HeroesRepository.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FirstSteps.RPG.Heroes;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+namespace FirstSteps.RPG
+{
+    public class HeroesRepository
+    {
+        const string FilePath = @"C:\Users\misza\source\repos\first-steps\src\FirstSteps\RPG\File\hero.json";
+
+        public static void SafeHero(Hero hero)
+        {
+            string serialize = JsonSerializer.Serialize(hero);
```

Comment:
You should serialize the `HeroModel` object and create it based on the hero. To achieve that you can add a new method in `Hero.cs` that return such an object. For example: `public HeroModel ToHeroModel() { ... }`.